problem/0000-rfc-template.md

```diff
@@ -0,0 +1,71 @@
+- Contribution Name: Thrift Support
+- Implementation Owner: Peter Novotank <peter.novotnak@reddit.com>, Linkerd2 Authors
+- Start Date: 6/3/2020
+- Target Date: 6/15/2020
+- RFC PR: [linkerd/rfc#9](https://github.com/linkerd/rfc/pull/9)
+- Linkerd Issue: [linkerd/linkerd2#0000](https://github.com/linkerd/linkerd2/issues/0000)
+- Reviewers: (Who should review the code deliverable? ex.@olix0r)
+
+# Summary
+
+[summary]: #summary
+
+Linkerd2-proxy and control plane support for the Thrift protocol.
+
+# Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+- Add protocol support to the linkerd2-proxy for at least TFramedTransport and TBinaryProtocol 
+- Support routing, retries and metrics for Thrift.
```

Comment:
How should routing work for thrift messages? For HTTP message, Linkerd will look at the host or authority header and if it is formatted like a kubernetes service (x.y.svc.z), it will route to that service.  Thrift messages don't encode any routing information in the message itself; how should we decide where to send them?